audiofeed-demo/src/main/kotlin/furhatos/demo/audiofeed/FurhatAudioFeedStreamer.kt

```diff
@@ -0,0 +1,61 @@
+package furhatos.monitor
+
+import org.zeromq.SocketType
+import org.zeromq.ZMQ
+import javax.sound.sampled.AudioFormat
+import kotlin.concurrent.thread
+
+class FurhatAudioFeedStreamer {
+
+    private val context: ZMQ.Context = ZMQ.context(1)
+    var running = false
+        private set
+    var runThread: Thread? = null
+    val audioFormat = AudioFormat(16000F, 16, 2, true, false)
```

Comment:
It might make sense to also create a sample that's a bit more specific as well. In this case, we're receiving two channel audio, one of which is what the robot hears, the other is what the robot says. I'd think that the majority of users will want to have only the "hearing" stream, not the "speaking" stream. The minimum sample should document which stream is which data, but it would be ideal to either let the user enroll in whichever (or both) stream. That's probably more work. Failing that, we might want to simply show them how to get the stream they want.